doc/users/prev_whats_new/whats_new_3.2.0.rst

```diff
@@ -0,0 +1,123 @@
+----------------------------------
+The *fname* argument to `FontProperties` can now be an `os.PathLike`\s
+e.g. ``FontProperties(fname=pathlib.Path("/path/to/font.ttf"))``.
+
+Gouraud-shading alpha channel in PDF backend
+--------------------------------------------
+The pdf backend now supports an alpha channel in Gouraud-shaded
+triangle meshes.
+
+Kerning adjustments now use correct values
+------------------------------------------
+Due to an error in how kerning adjustments were applied, previous versions of
+Matplotlib would under-correct kerning. This version will now correctly apply
+kerning (for fonts supported by FreeType). To restore the old behavior (e.g.,
+for test images), you may set :rc:`text.kerning_factor` to 6 (instead of 0).
+Other values have undefined behavior.
+
+.. plot::
+
+   import matplotlib.pyplot as plt
+
+   # Use old kerning values:
+   plt.rcParams['text.kerning_factor'] = 6
+   fig, ax = plt.subplots()
+   ax.text(0.0, 0.05, 'BRAVO\nAWKWARD\nVAT\nW.Test', fontsize=56)
+   ax.set_title('Before (text.kerning_factor = 6)')
+
+Note how the spacing between characters is uniform between their bounding boxes
+(above). With corrected kerning (below), slanted characters (e.g., AV or VA)
+will be spaced closer together, as well as various other character pairs,
+depending on font support (e.g., T and e, or the period after the W).
+
+.. plot::
+
+   import matplotlib.pyplot as plt
+
+   # Use new kerning values:
+   plt.rcParams['text.kerning_factor'] = 0
+   fig, ax = plt.subplots()
+   ax.text(0.0, 0.05, 'BRAVO\nAWKWARD\nVAT\nW.Test', fontsize=56)
+   ax.set_title('After (text.kerning_factor = 0)')
+
+
+bar3d lightsource shading
+-------------------------
+bar3d now supports lighting from different angles when the *shade* parameter is
+True, which can be configured using the ``lightsource`` parameter.
+
+Shifting errorbars
+------------------
+Previously, `plt.errorbar()` accepted a kwarg `errorevery` such that the
+command `plt.errorbar(x, y, yerr, errorevery=6)` would add error bars to
+datapoints `x[::6], y[::6]`.
+
+`errorbar()` now also accepts a tuple for `errorevery` such that
+`plt.errorbar(x, y, yerr, errorevery=(start, N))` adds error bars to points
+`x[start::N], y[start::N]`.
+
+Improvements in Logit scale ticker and formatter
+------------------------------------------------
+Introduced in version 1.5, the logit scale didn't have an appropriate ticker and
+formatter. Previously, the location of ticks was not zoom dependent, too many labels
+were displayed causing overlapping which broke readability, and label formatting
+did not adapt to precision.
+
+Starting from this version, the logit locator has nearly the same behavior as the
+locator for the log scale or the linear
+scale, depending on used zoom. The number of ticks is controlled. Some minor
+labels are displayed adaptively as sublabels in log scale. Formatting is adapted
+for probabilities and the precision is adapts to the scale.
```

Comment:
“the precision ~~is~~ adapts to the scale”?

Reply:
Fixed directly in @dstansby repository